homeassistant/components/verisure.py

```diff
@@ -150,20 +147,25 @@ def reconnect(self):
         if not self._lock.acquire(blocking=False):
             return
         try:
+            if self._password_retries < 0:
+                _LOGGER.error('Wrong password for Verisure, update config')
+                self._reconnect_timeout = time.time() + 60*60*24
```

Comment:
You're not resetting password retries here so it will keep hitting this block of code, why bother setting `reconnect_timeout` ?